plugins/modules/win_acl.py

```diff
@@ -45,6 +45,7 @@
       FileSystemRights U(https://msdn.microsoft.com/en-us/library/system.security.accesscontrol.filesystemrights.aspx).
     - If C(path) is a registry key, rights can be any right under MSDN
       RegistryRights U(https://msdn.microsoft.com/en-us/library/system.security.accesscontrol.registryrights.aspx).
+    - If I(path) is a certificate key, rights can be C(Read) and/or C(FullControl).
```

Comment:
Should state what version of the collection this was added in (1.1.0).

Reply:
Added comment about it being added in 1.8.0 (hopefully? 😄)